digdag-docs/src/operators/td.md

```diff
@@ -52,6 +60,28 @@ When you set those parameters, use [digdag secrets command](https://docs.digdag.
   td>: queries/step1.sql
   ```
 
+* **data**: query
+
+  A query template. This string can contain `${...}` syntax to embed variables.
+
+  Examples:
+
+  ```
+  td>:
+  data: "SELECT '${task_name}', * FROM nasdaq"
+  ```
+
+* **query**: query
+
+  A query. This string don't expand `${...}` syntax.
+
+  Examples:
+
+  ```
+  td>:
+  query: "SELECT * FROM nasdaq"
+  ```
+
```

Comment:
**query** parameter may expand variable as follows.
```
+task1:
  td>:
  query: "SELECT '${session_id}' FROM www_access limit 10"
```
log:
```
2019-08-09 19:11:53 +0900 [INFO] (0051@[0:yy_td]+test1+task1): Started presto job id=...:
SELECT '970' FROM www_access limit 10
```

Reply:
I updated the description.